hpx/util/detail/basic_function.hpp

```diff
@@ -27,6 +27,8 @@
 
 namespace hpx { namespace util { namespace detail
 {
+    static const std::size_t function_storage_size = 3*sizeof(void*);
```

Comment:
Why is this not constexpr?

Reply:
It wasn't before and there was no reason to add it now, other than perhaps a cosmetic one.